modules/core/src/main/java/org/apache/ignite/internal/processors/monitoring/GridMonitoringManager.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.monitoring;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.ignite.IgniteCheckedException;
+import org.apache.ignite.internal.GridKernalContext;
+import org.apache.ignite.internal.managers.GridManagerAdapter;
+import org.apache.ignite.internal.processors.monitoring.lists.MonitoringList;
+import org.apache.ignite.internal.processors.monitoring.lists.MonitoringListImpl;
+import org.apache.ignite.internal.processors.monitoring.sensor.SensorGroup;
+import org.apache.ignite.internal.processors.monitoring.sensor.SensorGroupImpl;
+import org.apache.ignite.spi.monitoring.MonitoringExposerSpi;
+
+/**
+ *
+ */
+public class GridMonitoringManager extends GridManagerAdapter<MonitoringExposerSpi> {
+    Map<MonitoringGroup, SensorGroup> sensors = new HashMap<>();
```

Comment:
Codestyle?

Reply:
Yes, of course.
Since it simple PoC I skip it for the sake of "fast development"